scripts/deploy.sh

```
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+REPO_URL="<https://github.com/awrion3/spring-roomescape-payment.git>"
+PROJECT_NAME="spring-roomescape-payment"
+BRANCH_NAME="step2"
+JAR_NAME="spring-roomescape-payment-0.0.1-SNAPSHOT.jar"
+
+echo "=== 1. Git Clone 또는 Pull ==="
+if [ -d "$PROJECT_NAME" ]; then
+  echo "기존 폴더 존재. 관련 프로세스 수행..."
+  cd "$PROJECT_NAME"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)